### PR TITLE
Removes EMAIL_ADDRESS action

### PR DESCRIPTION
**Pull request by CarinaWolli**
## What does this PR do?

Removes 'email to a specific email address' workflow action. 
Was originally implemented in #6075 but got reverted by #6085

### CODE REVIEW

File: packages/features/ee/workflows/lib/constants.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const WORKFLOW_TRIGGER_EVENTS = [
 export const WORKFLOW_ACTIONS = [
   WorkflowActions.EMAIL_HOST,
   WorkflowActions.EMAIL_ATTENDEE,
-  WorkflowActions.EMAIL_ADDRESS,
+  //WorkflowActions.EMAIL_ADDRESS, Disabling for now due to abuse episode
```
Comment (PeerRich) [inline packages/features/ee/workflows/lib/constants.ts:14]:
```suggestion
```

Comment (PeerRich) [reply packages/features/ee/workflows/lib/constants.ts:14]: not sure if // works. maybe just delete it?

Comment (JeroenReumkens) [reply packages/features/ee/workflows/lib/constants.ts:14]: It does work. But I agree that removing might make more sense. Adding it back later is easy.

**Review by JeroenReumkens — APPROVED** [review]

**Comment by PeerRich** [comment]
@CarinaWolli tests are failing because they rely on EMAIL_ADDRESS action. we'd need to remove that test, too

**Comment by CarinaWolli** [comment]
@PeerRich @JeroenReumkens I am filtering out the action now, I think that's an easier approach as otherwise, we have type errors

**Pull request merged**